### PR TITLE
tests: date range test

### PR DESCRIPTION
**Pull request by dinosk**
* ADDS test for date range aggregations

* FIXES conftest fixture indexing mock events

Signed-off-by: Dinos Kousidis <konstantinos.kousidis@cern.ch>

### CODE REVIEW

File: invenio_stats/contrib/aggregations/aggr-file-download/aggr-file-download-v1.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
       "numeric_detection": false,
       "properties": {
         "timestamp": {
-          "type": "date"
+          "type": "date",
+          "format": "date_optional_time"
```
Comment (nharraud) [inline invenio_stats/contrib/aggregations/aggr-file-download/aggr-file-download-v1.json]: We should keep the time, at least for now.

File: tests/conftest.py [hunk 2]
```diff
@@ -329,7 +340,7 @@ def sequential_ids():
     """Sequential uuids for files."""
```
Comment (nharraud) [inline tests/conftest.py]: You can add `scope="session"` to the `sequential_ids` fixture as it never changes.

File: invenio_stats/contrib/file-download/file-download-v1.json [hunk 3]
```diff
@@ -17,7 +17,8 @@
       "numeric_detection": false,
       "properties": {
         "timestamp": {
-          "type": "date"
+          "type": "date",
+          "format": "date_optional_time"
```
Comment (nharraud) [inline invenio_stats/contrib/file-download/file-download-v1.json]: We should keep the time, at least for now.

Comment (dinosk) [inline invenio_stats/contrib/file-download/file-download-v1.json]: Right, I changed it to `strict_date_hour_minute_second` in the event templates.

**Pull request merged**